### ISSUE TITLE
Access res/raw folder in android

### ISSUE THREAD

**Issue by popo63301** [wkh237/react-native-fetch-blob#602]
Hi everyone! I want to download sound from the web to store them in the res/raw folder for using it with react-native-sound. The problem is I don't have any idea on how to do it. Do you have any insight on this?

**Comment by isener** [comment]
hey friend, do you have any solution on it? I faced same problem though

**Comment by LaloHao** [comment]
Any update?

**Comment by popo63301** [comment]
@isener got something. I gave up because I found an other way to solve my problem, so I didn't need to continue looking for this solution.

**Comment by LaloHao** [comment]
Do you mind sharing? It may help us

**Comment by popo63301** [comment]
You gotta wait him answer

**Comment by isener** [comment]
you should try this pr https://github.com/itinance/react-native-fs/pull/270/files
It'll work with `CopyFileRes` method to devices externalstorage, then you get the url of that file and do what you want with that. For me, I'm using it with react-native-sound package and works well

**Comment by LaloHao** [comment]
Does this means that everytime you play the file is copied?

**Comment by isener** [comment]
nope, this means if you do something with image except play, you need to copy to local shared folder. For example sharing, cropping etc.
